projects/plugins/jetpack/class.jetpack-gutenberg.php

```diff
@@ -1072,15 +1072,24 @@ public static function set_availability_for_plan( $slug ) {
 		$is_simple_site = defined( 'IS_WPCOM' ) && IS_WPCOM;
 		$is_atomic_site = ( new Host() )->is_woa_site();
 
+		// keep track of specific features data per blog between calls.
+		static $site_specific_features = array();
+
 		// Check feature availability for Simple and Atomic sites.
 		if ( $is_simple_site || $is_atomic_site ) {
 
 			// Simple sites.
 			if ( $is_simple_site ) {
+				$current_blog_id = get_current_blog_id();
 				if ( ! class_exists( 'Store_Product_List' ) ) {
```

Comment:
Does it make sense to move this `require` to the level where we call `Store_Product_List` instead?

Reply:
Yeap, good point, updated.